doc/examples/effects/infinite_effects/lib/square.dart

```diff
@@ -6,16 +6,20 @@ import 'dart:ui';
 class Square extends PositionComponent {
   final Paint _paint;
 
-  Square(this._paint, double x, double y) {
+  Square(this._paint, double x, double y, {double angle = 0.0}) {
     width = 100;
     height = 100;
     this.x = x;
     this.y = y;
+    this.angle = angle;
     anchor = Anchor.center;
   }
 
   @override
   void render(Canvas canvas) {
+    canvas.translate(x, y);
+    canvas.rotate(angle);
```

Comment:
@spydon lets use `prepareCanvas` here them, instead of handle rotation manually.

Reply:
Oh I forgot to commit that, fixing when I get in from the park!

Reply:
Fixed